server/db/models/product.js

```diff
@@ -5,24 +5,24 @@ const Product = db.define('product', {
   name: {
     type: Sequelize.STRING,
     allowNull: false,
-    validate: {allowEmpty: false}
+    validate: {notEmpty: true}
   },
   description: {
     type: Sequelize.TEXT,
     allowNull: false,
-    validate: {allowEmpty: false, len: [5, 500]}
+    validate: {notEmpty: true, len: [5, 500]}
   },
   stock: {
     type: Sequelize.INTEGER,
     allowNull: false,
-    validate: {allowEmpty: false, min: 0}
+    validate: {isNumeric: true, min: 0}
   },
   price: {
-    type: Sequelize.DECIMAL,
+    type: Sequelize.INTEGER,
```

Comment:
Why is this Integer instead of Decimal now?

Reply:
Based on the code review with Ben yesterday on storing currency as integers